test/unit/plugins/rest_spec.js

```diff
@@ -131,40 +147,180 @@ describe("The Rest plugin", function () {
 			});
 		});
 
-		describe("when there is a problem with the database connection", function () {
-			var result;
+	});
+
+	describe("creating a new instance", function () {
```

Comment:
How is this different that the `describe` block above (the description looks the same)?